tests/__init__.py

```diff
@@ -0,0 +1 @@
+# dummy file to init the directory
```

Comment:
It is common practice to leave this empty. It should get rid of the codacy warning as well.

Reply:
Thanks. Done. :-)